ibverbs/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "ibverbs"
+version = "0.4.4"
+
+description = "Bindings for RDMA ibverbs through rdma-core"
+readme = "README.md"
+authors = [
+    "Jon Gjengset <jon@thesquareplanet.com>",
+    "Jonathan Behrens <fintelia@gmail.com>"
+]
+edition = "2018"
+
+homepage = "https://github.com/jonhoo/rust-ibverbs"
+repository = "https://github.com/jonhoo/rust-ibverbs.git"
+documentation = "https://docs.rs/ibverbs/"
+
+keywords = ["rdma", "ibverbs", "networking", "ffi"]
+categories = ["network-programming", "api-bindings"]
+
+license = "MIT/Apache-2.0"
+
+[badges]
+travis-ci = { repository = "jonhoo/rust-ibverbs" }
+maintenance = { status = "looking-for-maintainer" }
```

Comment:
_nudge nudge_

Reply:
Oh right! I was gonna ask about that. I'd honestly be glad to help maintain!

Reply:
That's awesome! Let's work our way through this PR, and then chat about it afterwards? This crate is fairly low maintenance in the first place, but it could certainly use some love (like this PR) :)